Cargo.toml

```diff
@@ -40,6 +41,7 @@ tokio = { version = "1.3", features = ["net", "io-util", "rt", "sync", "macros",
 tokio-rustls = { version = "0.22", optional = true }
 uuid = "0.8.1"
 webpki = { version = "0.21", optional = true }
+serde = { version = "1", optional = true }
```

Comment:
This is a dependency because cdrs_orm can map serializable types into the db back and forth, without any hassle for the user